test/server/reporter-test.js

```diff
@@ -39,20 +39,38 @@ describe('Reporter', function () {
 
     var testMocks = [
         {
-            name:               'fixture1test1',
-            fixture:            fixtureMocks[0],
-            skip:               false,
-            screenshotExpected: true,
-            meta:               {
+            name:        'fixture1test1',
+            fixture:     fixtureMocks[0],
+            skip:        false,
+            screenshots: [{
+                screenshotPath:      'screenshot1.png',
+                thumbnailPath:       'thumbnail1.png',
+                userAgent:           'chrome',
+                onFail:              false,
+                quarantineAttemptID: 2
```

Comment:
`quarantineAttemptID` -> `quarantineAttempt` - isn't it just a count, not an ID?

Reply:
not a count, ID

Reply:
okaaaay